modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -217,6 +217,18 @@ This feature requires mounting the kernel debug filesystem, so the eBPF agent po
 It requires using the OVN-Kubernetes network plugin with the Observability feature.
 IMPORTANT: This feature is available as a Developer Preview. +
 
+- `PacketTranslation`: enable enriching flows with packet's translation information.  +
+
+- `EbpfManager`: allow using eBPF manager to manage netobserv ebpf programs.  +
+
+IMPORTANT: This feature is available as a Developer Preview. +
+
+- `UDNMapping`, to enable interfaces mappind to udn.  +
+
+This feature requires mounting the kernel debug filesystem, so the eBPF agent pods have to run as privileged.
```

Comment:
Per ISG: [Use have to only when must is too restrictive.](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#h)
```suggestion
This feature requires mounting the kernel debug filesystem, so the eBPF agent pods must run as privileged.
```